src/Apps/Order/Components/__tests__/BankAccountPicker.jest.tsx

```diff
@@ -205,7 +205,7 @@ describe("BankAccountFragmentContainer", () => {
         }),
       })
       const page = new BankAccountPickerTestPage(wrapper)
-      page.clickRadio(3)
+      page.clickRadio(2)
```

Comment:
Resolves `Error: Method “simulate” is meant to be run on 1 node. 0 found instead.`